<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PogadeMainWindow</class>
 <widget class="QMainWindow" name="PogadeMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>699</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pogade</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>artwork/appicons/pogade-logo.png</normaloff>artwork/appicons/pogade-logo.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>699</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNewProject"/>
    <addaction name="actionOpenProject"/>
    <addaction name="actionRecent"/>
    <addaction name="separator"/>
    <addaction name="actionCloseProject"/>
    <addaction name="separator"/>
    <addaction name="actionExportProject"/>
    <addaction name="actionImportProject"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAll"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionSettings"/>
    <addaction name="separator"/>
    <addaction name="actionProjectViewer"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNewProject">
   <property name="icon">
    <iconset resource="appicons.qrc">
     <normaloff>:/appicons/document-new</normaloff>:/appicons/document-new</iconset>
   </property>
   <property name="text">
    <string>New Project</string>
   </property>
   <property name="toolTip">
    <string>Create a new project</string>
   </property>
  </action>
  <action name="actionOpenProject">
   <property name="icon">
    <iconset resource="appicons.qrc">
     <normaloff>:/appicons/document-open</normaloff>:/appicons/document-open</iconset>
   </property>
   <property name="text">
    <string>Open Project</string>
   </property>
   <property name="toolTip">
    <string>Open an existing project</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="appicons.qrc">
     <normaloff>:/appicons/window-close</normaloff>:/appicons/window-close</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>Exit Pogade</string>
   </property>
  </action>
  <action name="actionCloseProject">
   <property name="text">
    <string>Close Project</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="appicons.qrc">
     <normaloff>:/appicons/document-save</normaloff>:/appicons/document-save</iconset>
   </property>
   <property name="text">
    <string>Save Project</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset resource="appicons.qrc">
     <normaloff>:/appicons/document-save-as</normaloff>:/appicons/document-save-as</iconset>
   </property>
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionProjectViewer">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Project Viewer</string>
   </property>
  </action>
  <action name="actionExportProject">
   <property name="text">
    <string>Export Project</string>
   </property>
  </action>
  <action name="actionCodeEditor">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Code Editor</string>
   </property>
  </action>
  <action name="actionSaveAll">
   <property name="text">
    <string>Save All</string>
   </property>
  </action>
  <action name="actionImportProject">
   <property name="text">
    <string>Import Project</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="appicons.qrc">
     <normaloff>:/appicons/help-content</normaloff>:/appicons/help-content</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="appicons.qrc">
     <normaloff>:/appicons/help-about</normaloff>:/appicons/help-about</iconset>
   </property>
   <property name="text">
    <string>About...</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionRecent">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="appicons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>PogadeMainWindow</receiver>
   <slot>exitProgram()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionNewProject</sender>
   <signal>triggered()</signal>
   <receiver>PogadeMainWindow</receiver>
   <slot>newProject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSave</sender>
   <signal>triggered()</signal>
   <receiver>PogadeMainWindow</receiver>
   <slot>saveProject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOpenProject</sender>
   <signal>triggered()</signal>
   <receiver>PogadeMainWindow</receiver>
   <slot>openProject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExportProject</sender>
   <signal>triggered()</signal>
   <receiver>PogadeMainWindow</receiver>
   <slot>exportProject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCloseProject</sender>
   <signal>triggered()</signal>
   <receiver>PogadeMainWindow</receiver>
   <slot>closeProject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSaveAll</sender>
   <signal>triggered()</signal>
   <receiver>PogadeMainWindow</receiver>
   <slot>saveProjectAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionImportProject</sender>
   <signal>triggered()</signal>
   <receiver>PogadeMainWindow</receiver>
   <slot>importProject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAbout</sender>
   <signal>triggered()</signal>
   <receiver>PogadeMainWindow</receiver>
   <slot>showAbout()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionHelp</sender>
   <signal>triggered()</signal>
   <receiver>PogadeMainWindow</receiver>
   <slot>showHelp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSettings</sender>
   <signal>triggered()</signal>
   <receiver>PogadeMainWindow</receiver>
   <slot>showSettings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRecent</sender>
   <signal>triggered()</signal>
   <receiver>PogadeMainWindow</receiver>
   <slot>loadRecentProject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>exitProgram()</slot>
  <slot>newProject()</slot>
  <slot>openProject()</slot>
  <slot>closeProject()</slot>
  <slot>saveProject()</slot>
  <slot>exportProject()</slot>
  <slot>saveProjectAll()</slot>
  <slot>importProject()</slot>
  <slot>showAbout()</slot>
  <slot>showHelp()</slot>
  <slot>showSettings()</slot>
  <slot>loadRecentProject()</slot>
 </slots>
</ui>
